happy-birthday-to-you.js

```diff
@@ -0,0 +1,153 @@
+/* 
```

Comment:
This is not as important, but it is good to start early. For JavaScript, convention is to use 2 spaces for indentation. On sublime, you can set syntax specific settings and you can set the indentation to 2, and spaces_for_indentation to be true. Let me know if you have trouble with this. You can continue to use the tab button, but it will just make /s/s rather than /t in the editor. This is good because the code will then look consistent across all machines since a /s is always a /s. But a /t can be different depending on a person's setup.